test/case/infix_dhcp/dhcp_basic/topology.dot

```diff
@@ -1,24 +1,23 @@
 graph "1x2" {
 	layout="neato";
 	overlap="false";
-	esep="+20";
+	esep="+80";
 
         node [shape=record, fontname="monospace"];
 	edge [color="cornflowerblue", penwidth="2"];
 
-	host [
```

Comment:
This is also a little strange, it is still the host. I think we should stick with calling it host.

Reply:
yes, I agree. I'll revert to 'host' :)